### ISSUE TITLE
[Accessibility][Screen Reader - ConetxualMenu]: Screen reader does not announce the state as Expand/Collapsed for the "New" menu item.

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#8135]
**Test environment: **
Platform: Windows 10
OS Build : Build 18334.19h1_release.190205-1505
Browser: Google Chrome Version: 72.0.3626.109
Screen reader: JAWS
 
**Repro steps:**
1. Open [URL](https://developer.microsoft.com/en-us/fabric#/components).
2. Navigate to "Basic Imputs" button in left navigation pane and activate it.
3. Navigate to "ContextualMenu" button and activate it.
4. Navigate to "Click for ContextualMenu" button under "ContextualMenu with submenus".
5. Navigate the New menu item.
6. Observe the issue.
 
**Actual result: **
Screen Reader is announcing menu items as "sub-menu" instead of the state "Expand/Collapsed" for the " New, Share, Share /w Split" menu items.

**Expected result:**
Screen Reader should announce the state of the control properly as Expand/collapsed.

**Observation:**
Same issue repro with NVDA+Chrome

**Attachment:**
[Bug_8135_1](https://microsoft-my.sharepoint.com/:v:/g/personal/v-shshu_microsoft_com1/Ee9JGJZK1ZRMmBNj4dj_dpYBMlj0f9_BsU15C4fiMnsEUg?e=LzYlMa); [Bug_8135_2](https://microsoft-my.sharepoint.com/:i:/g/personal/v-shshu_microsoft_com1/ER2KxQDj_ERIuFT1yYFOEpQBjrOSX-BOp4tqe5rdGjRtVg?e=fessFQ); [Bug_8135_3](https://microsoft-my.sharepoint.com/:i:/g/personal/v-shshu_microsoft_com1/ESwvteOkdYpNuEFKnVbEBFcBgg4jRD7iQ-4NphSG0GMSjQ?e=V4bJBh); [Bug_8135_4](https://microsoft-my.sharepoint.com/:i:/g/personal/v-shshu_microsoft_com1/EWW2_kpXn11HkEb14fVDMWQB9RiVgBBwXEPOATgFqtGNiQ?e=MRvKb5).
@micahgodbolt

Tags:
HCL-MSTeams; Shifts; HCL-Fabric; A11yMAS; MAS4.2.1; Web-Chrome.

**Comment by leddie24** [comment]
Hi @TestAccessibilityGIT , thanks for submitting this issue.  This seems to be working fine with Edge/Narrator, but it looks like it's not functioning the same in Chrome/NVDA/JAWS.  I will get in touch with the Narrator/Edge team to see what screen reader and/or browser is breaking accessibility standards.

**Comment by micahgodbolt** [comment]
Can we tripple check if this/is not working in Edge/Narrator? Seems the two of you are getting different results.

**Comment by leddie24** [comment]
This is working in narrator/edge for me.  Here's the output from Narrator Buddy

```
New , collapsed, menu item,
<say-as interpret-as="cardinal">1</say-as> of <say-as interpret-as="cardinal">3</say-as>, 
Share , collapsed, menu item,
<say-as interpret-as="cardinal">2</say-as> of <say-as interpret-as="cardinal">3</say-as>, 
```

**Comment by leddie24** [comment]
Getting in touch with the right contacts on JAWS team to file and triage this issue on their end currently.

**Comment by awalia13** [comment]
Logged Bug with JAWS Team.
